docs/proc/how_we_work_in_the_team.md

```diff
@@ -0,0 +1,30 @@
+# Bref
+Here you can find how we are working in the dev team and what dev team members
+should to help other members be on the same page.
+
+# Code contribution guide
+Please find and read document `docs/proc/code_contribution_guide.md`.
+
+# Releases
+Each repository SHOULD have a release tag, and each tag should have a strong formatted text, which is: MAJOR-MINOR-PATCH. For forked repos that we are changing SHOULD be: FORKED_REPOS_MAJOR-FORKED_REPOS_MINOR-FORKED_REPOS_PATCH-MAJOR-MINOR-PATCH.
+
+Example: let's say polkadotsdk has version 1.0.0, qf-solochain has version 0.5.0, then the full version is 1.0.0-0.5.0.
+
+# Daily status reports
+Each team member responsible for development SHOULD post a daily status report to related zulip channel. Key ideas behind this report is reduce number of calls and keep business team in the same page with dev team. However dev team members should keep GH's tickets upated no matter on this report.
+
+## Template Daily status report
+
+### Case: NAME OF THE CASE (link to GH ticket)
+### Experment: NAME OF THE EXPEREMENT (link to GH ticket)
+### List of tickets for today:
+- [NAME OF THE TICKET] (link to GH ticket), estimation to done: DAYS, current status: A LINK TO LATEST GH comment (preferred) or TEXT - -
```

Comment:
> estimation to done: DAYS

Should we post same estimation on the daily basis?

What should be the line for completed ticket?

Reply:
Good catch, I think MM:DD would be better, or just add this info into the GH ticket, and do not post it into the status.

Reply:
Date and updating GH ticket both looks good for me